<?php
   $title = "Selected projects for FAIRification 2019 | FAIRplus";
   $description = "Selected projects for FAIRification";
   $section = "choosing-data";
   $page = "datasets-2019";
   include $_SERVER['DOCUMENT_ROOT']."/includes/header.inc";
?>

    <main id="main-content">
       <div class="inner-container">
          <div class="flex-container">
             <div class="percent-75">
                <article>
                  <h1>Datasets 2019</h1>
                  <p>In 2019, <strong>datasets from four IMI projects</strong> were selected for early FAIRification: </p>
                  <ul class="low-padding">
                    <li><a href="#nd4bb">ND4BB-TRANSLOCATION</a></li>
                    <li><a href="#oncotrack">Onco Track</a></li>
                    <li><a href="#etox">eTOX</a></li>
                    <li><a href="#resolute">RESOLUTE</a></li>
                  </ul>
                  <p>These projects represent a different indication area, a different stage of the drug discovery process, and a different IMI project maturity level. The FAIRified data from these projects is now available in the <a href="http://datacatalog.elixir-luxembourg.org/">IMI FAIR Data Catalog</a>.</p>
                  <p>These were the first results of the FAIRplus Squad teams, who worked on the FAIRifying this data throughout 2019. Based on their experience, they developed a <a href="https://drive.google.com/file/d/1P2MJSLIST6AJyhmFdIKRfb_Xaqvb9TXX/view">detailed FAIRification process</a> which will be used in 2020 and beyond to FAIRify further datasets.</p>

                  <h2 class="x-top-margin" id="nd4bb">ND4BB-TRANSLOCATION </h2>
                  <p><a href="http://www.nd4bb.eu"><img src="/images/logos/nd4bb.png" class="float-right" width="300"></a>The ND4BB-TRANSLOCATION IMI consortium (<a href="http://www.nd4bb.eu">www.nd4bb.eu</a>) operated between 2013 and 2018 and involved 5 EFPIA and 20 Public partners. Their goal was to develop new insights into the molecular determinants of antibiotic drug influx and efflux in gram negative bacteria. They also developed a platform (the ND4BB InfoCentre) to facilitate information sharing in antibiotic research. </p>
                  <p>Within the InfoCentre, data on historical successes and failures in antibacterial research and development are combined with information drawn from public databases and new insights generated within the ND4BB consortia on the fundamental biophysical determinants of compound transport mechanisms.</p>
                  <p> The IMI InfoCentre was developed to coordinate the disclosure and combined analyses of previously confidential information. This information was provided primarily by participating organisations across the new Drug for Bad Bugs program. Initial FAIRification efforts focused on a highly annotated database covering the antimicrobial and transport related properties of clinical antibiotics and tool compounds. Additional data sets cover PK and efficacy studies for antibiotics from EFPIA partners.</p>
                  <h2  class="x-top-margin" id="oncotrack">Onco Track</h2>
                  <p><a href="http://www.oncotrack.eu">www.oncotrack.eu<img src="/images/logos/oncotrack.jpg" class="float-right" width="300"></a>The OncoTrack consortium (<a href="http://www.oncotrack.eu">www.oncotrack.eu</a>) was active from 2011 to the 2016 with 8 EFPIA and 11 Public partners. The aim of the project was to establish innovative approaches for biomarker assessment in the peripheral circulation in order to assess the potential of circulating tumor cells, cancer stem cells and/or nucleic acids as a surrogate for invasive biopsies. </p>
                  <p>OncoTrack was the first large study to provide extensive molecular characterization of a patient donor cohort, encompassing all the disease stages and to include the establishment of matched in vivo and in vitro models. Data sets for FAIRification included whole exome (in some cases whole genome) sequencing, transcript sequencing and methylome analysis of clinical tumour samples. Confirmatory genome sequencing and transcriptome analyses were performed on xenograft and cell culture models. </p>
                  <p>In addition, drug response data for a panel of 16 therapeutic agents in the xenograft and cell culture models was generated. Related proteome data is available from both multiplex-MS methods and RPA studies. Possible scientific and societal impact from systematic reuse of the OncoTrack data includes the potential for development of new diagnostic procedures and the wider implementation of the developed biological models in the translational research process.
                  </p>
                  <h2  class="x-top-margin" id="etox">eTOX</h2>
                  <p><a href="http://www.etoxproject.eu"><img src="/images/logos/etox.png" class="float-right" width="200"></a>The eTOX consortium (<a href="http://www.etoxproject.eu">www.etoxproject.eu</a>) was composed of 17 Public and 13 EFPIA partners and was active from 2010 to 2016. The focus of the consortia was to create a platform for legacy data sharing from industrial and public sources in order to improve the efficiency of drug safety testing. Of special interest was the wealth of the high quality toxicology data in the archives of the pharmaceutical companies, which through the development of innovative strategies and novel software tools could be interrogated to enable an early prediction of the potential side-effects of new drug candidates on the basis of integrative approaches.
                  </p>
                  <p>In practice, this led to the development of a large number of models by diverse organizations, their integration into a single system (eTOXsys) and their adoption by the pharmaceutical industry. Strict procedures and tools were established for enabling the harmonized development, documentation, verification and implementation of the models, as well as their versioning and maintenance. The project developed new tools including eTOXlab, a flexible modelling framework, ADAN, a method for the assessment of the model applicability domain, and specific protocols for model documentation (OECD QSAR). </p>
                  <p>In the context of the FAIRplus pilot, a test database covering compound toxicological data from around 100 compounds will be analysed. This will act as a starting point for the eventual FAIRification of the entire eTOX data resource which covers over 8.8 million pre-clinical data points from 8,196 pre-clinical studies on nearly 2000 chemicals. Ultimately, it is planned that these FAIR data approaches will be integrated into the data management procedures of the successor (<a href="http://www.etransafe.eu">IMI project, eTRANSAFE </a>).</p>
                  <h2 class="x-top-margin" id="resolute">RESOLUTE</h2>
                  <p><a href="http://www.re-solute.eu"><img src="/images/logos/resolute.png" class="float-right" width="300"></a>RESOLUTE is a public-private partnership (<a href="http://www.re-solute.eu">www.re-solute.eu</a>) which started in 2018, with 13 partners from academia and industry. The consortia aims to trigger an escalation in the appreciation and intensity of research on solute carriers (SLCs) and to establish SLCs as a tractable target class for medical research and development. </p>
                  <p>Through the coupling of an inclusive, open-access ethos to the data, results, techniques and reagents with the highest-possible quality of research output, RESOLUTE expects to accelerate the pace and increase the impact of research in the field of SLCs to the global benefit of basic academic research through to applied research in SMEs and pharmaceutical companies. Data types will include RNA-Seq and quantitative gene expression analyses of various cell lines and SLC-target family wide cell viability screen measurements using sgRNA libraries. </p>
                  <p>As a newly commenced project the RESOLUTE consortium, through its cooperation with FAIRplus, will seek to implement FAIR methods into its data management approach from the earliest stages of its operation.</p>

                  <ul class="inline-links x-top-margin">
                  <li><a href="/choosing-data/">Choosing a dataset</a></li>
                  <li><a href="#main-header">Back to the top</a></li>
                  </ul>

                </article>
             </div>
             <div class="percent-25">
               <?php
                 include $_SERVER['DOCUMENT_ROOT']."/includes/subnav-choosing-data.inc";
               ?>
               <?php
                  include $_SERVER['DOCUMENT_ROOT']."/includes/stay-in-touch.inc";
               ?>
             </div>
          </div>
       </div>
    </main>
<?php
  include $_SERVER['DOCUMENT_ROOT']."/includes/footer.inc";
?>
